docs/users_guide/glasgow_exts.rst

```diff
@@ -12461,7 +12465,6 @@ above limitations induce no loss of generality: ::
     import GHC.StaticPtr
 
     data Dict c = c => Dict
-      deriving Typeable
```

Comment:
Again, here I'm modifying example code to remove a `Typeable` derivation that has no effect.